src/libs/dbHandler.go

```diff
@@ -457,7 +457,6 @@ func upsertSysSummarySQL(db *sql.DB, summary types.SystemSummary, timeCount type
 		summary.PodName,
 		summary.Operation,
 		summary.Labels,
-		summary.Deployment,
```

Comment:
Let's keep the deployment related code in this file too. Please undo `summary.Deployment` changes from this file.